landbot-org/eslint-config
# feat: add sample project and improve eslint config

Changes:

- Add `sample` project to test `eslint` and `prettier` rules
- Improve `release` workflow
- Add `build` workflow to test rules as a requirement/check when submitting a PR
- Update `prettier` sort imports rule to set react related imports at the top and styles at the end
- Update `semantic-release `version
- `eslint`
  - Add `no-default-export` rule
  - Add rule to avoid `import React from 'react';` as from react 17 is unnecessary
  - Add folder naming convention (`kebab-case`)
